client/ContributionsListView.js

```diff
@@ -1,20 +1,22 @@
 import Button from '@material-ui/core/Button';
+import { Divider } from '@material-ui/core';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import React, {Component} from 'react';
 import {createMuiTheme, MuiThemeProvider, withStyles} from '@material-ui/core/styles';
 import Paper from "@material-ui/core/Paper";
+import {Add, Edit, Visibility } from "@material-ui/icons";
 
 
 import fb from './firebase';
 
-const theme = createMuiTheme({
-    palette: {
-        primary: {
-            main: '#c51162',
-        },
-    },
-});
+// const theme = createMuiTheme({
+//     palette: {
+//         primary: {
+//             main: '#c51162',
+//         },
+//     },
+// });
```

Comment:
So this is how we used to do themes...
Are we still going to need the commented code?